msteams-platform/concepts/build-and-test/apps-package.md

```diff
@@ -6,51 +6,55 @@ ms.topic: conceptual
 
 # Create an app package for your Microsoft Teams app
 
-Apps in Teams are defined by an app manifest JSON file, and bundled in an app package with their icons. You'll need an app package to upload and install your app in Teams and publish to either your Line of Business app catalog or to AppSource.
+This document guides you on how to create your app package.
+
+Apps in Teams are defined by an app manifest JSON file and bundled in an app package with their icons. You need an app package to upload and install your app in Teams and publish to an App Store or to AppSource.
 
 A Teams app package is a .zip file containing the following:
 
-* A manifest file named `manifest.json`, which specifies attributes of your app and points to required resources for your experience, such the location of its tab configuration page or the Microsoft app ID for its bot.
+* A manifest file named **manifest.json**, which specifies attributes of your app and points to needed resources for your experience. For example, location of the tab configuration page or the Microsoft app ID for the bot.
 * [Color and outline icons for your app](#app-icons).
 
-## Creating a manifest
+## Create a manifest
```

Comment:
```suggestion
## App manifest
```
At this point, your manifest has been created. I think maybe we only need to briefly describe what the manifest is (your app configurations) and how it's part of your package.